readcourse.data

```diff
@@ -0,0 +1,148 @@
+import json
+import os
+from pymongo import MongoClient
+
+class Course:
+	def __init__(self, crn):
+		self.crn = crn
+
+	def __init__(self, **kwargs, *args):
+		self.UID = kwargs.['UID']
+		self.crn = kwargs.['crn']
+		self.course_num = kwargs.['course_num']
+		self.section_num = kwargs.['section_num']
+		self.campus = kwargs.['campus']
+		self.num_credit = kwargs.['num_credit']
+		self.course_title = kwargs.['course_title']
+		self.days = kwargs.['days']
+		self.startTime = kwargs.['startTime']
+		self.endTime = kwargs.['endTime']
+		self.cap = kwargs.['cap']
+		self.wl_cap = kwargs.['wl_cap']
+		self.instructor_name = kwargs.['instructor_name']
+		self.startDate = kwargs.['startDate']
+		self.endDate = kwargs.['endDate']
+		self.location = kwargs.['location']
+		self.attribute = kwargs.['attribute']
+    
+//
+		self.lab = list()
+		self.act = list()
+		self.wl_act = list()
+
+class Instructor:
+	def __init__(self, firstName, middleName, lastName):
+		self.firstName = firstName
+		self.middleName = middleName
+		self.lastName = lastName
+		self.website = ''
+		self.email = ''
+		self.department = list()
+		self.relatedCourse = dict()
+
+class Department:
+	def __init__(self, deptName):
+		self.deptName = deptName
+		self.courses = list()
+
+class Lab:
+	def __init__(self, UID, days, startTime, endTime, \
+				startDate, endDate, instructor, location):
+		self.UID = UID
+		self.days = days
+		self.startTime = startTime
+		self.endTime = endTime
+		self.startDate = startDate
+		self.endDate = endDate
+		self.instructor = instructor
+		self.location = location
+
+
+
+def main():
+
+	script_dir = os.path.dirname(__file__)
+	rel_path = "2018_Winter_De_Anza.json"
+	filename = os.path.join(script_dir, rel_path)
+
+	if filename:
+		with open(filename, 'r') as f:
+			course_raw_data = json.load(f)
+
+	course_list = []
+	department_list = []
+	total_course = 0
+	for department in course_raw_data["2018 Winter De Anza"]["CourseData"]:
```

Comment:
should not hard code here

Reply:
Do you mean change ["2018 Winter De Anza"]["CourseData"] to a relative one?

Reply:
we should not hard code `2018 Winter De Anza`